ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -125,8 +125,26 @@ function is_openshift_version_gte() {
     printf '%s\n%s' "$1" "${DS_OPENSHIFT_VERSION}" | sort -C -V
 }
 
+function extract_oc_for_upgrade() {
+  # shellcheck disable=SC2087
+    ssh "${SSHOPTS[@]}" "root@${IP}" bash -x - << EOF
+EXTRACTDIR=\$(mktemp --tmpdir -d "installer-upgrade-XXXXXXXXXX")
+
+EXTRACTNEWOC="oc adm release extract --registry-config ${DS_WORKING_DIR}/pull_secret.json \
+   --command=oc --to "\${remote_extract_dir}" $1"
+
+\$EXTRACTNEWOC
+
+sudo mv "\${remote_extract_dir}/oc" /usr/local/bin
```

Comment:
mv: cannot stat '/oc': No such file or directory
going to figure out this part 😅

Reply:
https://storage.googleapis.com/test-platform-results/pr-logs/pull/openshift_release/58754/rehearse-58754-periodic-ci-openshift-release-master-nightly-4.17-upgrade-from-stable-4.16-e2e-metal-ipi-ovn-upgrade/1864500724377849856/build-log.txt